src/math.ts

```diff
@@ -1080,6 +1080,15 @@ export class Polygon {
 	}
 }
 
+export function evaluateBezier(pt1: Vec2, pt2: Vec2, pt3: Vec2, pt4: Vec2, t: number) {
```

Comment:
Thinking about the best name here, what do you think about `generateBezier()`, or `genBezier()`? I'm kinda towards the later

Reply:
It literally evaluates the cubic Bezier function at the point t in t-space. It doesn't generate anything other than the point at the given t. I have other functions which evaluate the first derivative (velocity on the Bezier at t, to for example set angle so that you turn along the path), the second derivative (acceleration on the Bezier at t, to animate a thruster for example) or an adapter which evaluates the Bezier in uniform s space, to keep velocity constant, or get fixed offsets along the Bezier or other spline.
genBezier, or better genSpline would be a function which does what goes on partly in drawSpline, evaluating the function in n points to get a list of points representing the spline.

Reply:
Ok that makes sense 👍